alignak/http/daemon.py

```diff
@@ -46,6 +46,51 @@
 logger = logging.getLogger(__name__)  # pylint: disable=C0103
 
 
+try:
+    class Pyopenssl(pyOpenSSLAdapter):
+        """
+        Use own ssl adapter to modify ciphers. This will disable vulnerabilities ;)
+        """
+
+        def __init__(self, certificate, private_key, certificate_chain=None, dhparam=None):
+            """
+            Add init because need get the dhparam
+
+            :param certificate:
+            :param private_key:
+            :param certificate_chain:
+            :param dhparam:
+            """
+            super(Pyopenssl, self).__init__(certificate, private_key, certificate_chain)
+            self.dhparam = dhparam
+
+        def get_context(self):
+            """Return an SSL.Context from self attributes."""
+            cont = SSL.Context(SSL.SSLv23_METHOD)
+
+            # override:
+            ciphers = (
+                'ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+HIGH:'
+                'DH+HIGH:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+HIGH:RSA+3DES:!aNULL:'
+                '!eNULL:!MD5:!DSS:!RC4:!SSLv2'
+            )
+            cont.set_options(SSL.OP_NO_COMPRESSION | SSL.OP_SINGLE_DH_USE | SSL.OP_NO_SSLv2 |
+                             SSL.OP_NO_SSLv3)
```

Comment:
Previously it was : SSL.OP_NO_SSLv2 | SSL.OP_NO_SSLv3 | SSL.OP_NO_TLSv1 | SSL.OP_NO_TLSv1_1 |        SSL.OP_NO_COMPRESSION. Why allowing TLSv1 and TLSv1.1 ? What is SINGLE DH ?